.gitignore

```diff
@@ -3,3 +3,4 @@ elo.db
 *.pyc
 .DS_Store
 .*.swp
+/venv
```

Comment:
there should still be no `/venv`

Reply:
What? Why not? I always considered a .gitignore to be a kind of an informal thing that doesn't really need regulation unless it becomes an issue; am I wrong?